device_plugins/deploy_dsa.md

```diff
@@ -0,0 +1,54 @@
+# Create Intel DSA Device Plugin CR
+
+## Create a CR via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Open **Intel Device Plugins Operator**.
+3.	Navigate to tab **Intel DSA Device Plugin**.
+4.	Click **Create DSADevicePlugin** -> set correct parameters -> Click **Create** 
+5.	Optional: If you want to make any customizations, select YAML view and edit the details. When you are done, click **Create**.
+
+## Verify via web console
+1.	Verify CR by checking the status of **Workloads** -> **DaemonSet** -> **intel-dsa-plugin**.
+2.	Now `DsaDevicePlugin` is created.
+
+## Create CR via CLI
+Apply the CR yaml file:
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/device_plugins/dsa_device_plugin.yaml
+```
+
+## Verify via CLI
+Verify that the device plugin CR is ready: 
+```
+$ oc get DsaDevicePlugin
+```
+Output: 
+```
+NAME		        DESIRED		READY	NODE SELECTOR	                                    AGE
+dsadeviceplugin-sample  3 	        3       {"intel.feature.node.kubernetes.io/dsa":"true"}     98m
+```
+
+# Verify DSA Device Plugin 
+After the plugin is deployed, use below command to verify DSA resources: 
+```
+$ oc describe node srf-2 | grep dsa.intel.com
+  dsa.intel.com/wq-user-dedicated:  0
+  dsa.intel.com/wq-user-shared:     160
+  dsa.intel.com/wq-user-dedicated:  0
+  dsa.intel.com/wq-user-shared:     160
+ ```
+
+## DSA Resource Configuration
+By default the DSA plugin uses [this configuration file](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/demo/dsa.conf).
```

Comment:
Could you briefly explain the DSA resource Configuration here, and also add the link point to the detailed explaining?

Reply:
I agree it makes sense to add a bit more on _how_ the configuration is done. @mregmi has some basics already covered . Essentially, the initcontainer just comes with `accel-config` tool that is used to configure the DSA WQs. The "default" setting creates dedicated WQs for each DSA device so that it's four groups per device where each groups is with 1 WQ linked to 1 (dedicated) engine (i.e., each k8s resource is "equal").

I'd also add tha node specific configs are allowed. The default config can be overridden using the configmap where the entries are `dsa.conf` or `dsa-<nodename>.conf`(or both).

At the moment `idxd-init.sh` only assumes that each physical DSA device is configured the same way...

Reply:
i will add more details here. Thanks

Reply:
added thanks

Reply:
@mythi  could you please help review this updated information about the resource configuration?

Reply:
applied all suggestions